front/src/HOC/withLogger.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+
+const getDisplayName = SomeComponent => {
+  return (
+    SomeComponent.displayName || SomeComponent.name || 'Component'
+  );
+};
+
+export const withLogger = (params = {}) => {
+  const { output, lifecycles = [] } = params;
+  const logProps = lifecycles.findIndex(el => el === 'props') !== -1;
```

Comment:
Can we apply another method here to improve readability?

Reply:
indexOf ?